lib/ExtUtils/Liblist/Kid.pm

```diff
@@ -96,6 +96,8 @@ sub _unix_os2_ext {
             next;
         }
 
+        next unless defined $thislib;
```

Comment:
This doesn't make a lot of sense to me. The only possible way for $thislib to be undefined here is if quotewords returned an undefined argument (in which case the first substitution would warn before this line) or if it's assigned undef from catdir (does not seem possible) or `qq{"$thislib"}` (not possible).

Reply:
Indeed your first warning is from the initial substitution which means quotewords returned an `undef` for some reason.